tests/integration/test_transformers.py

```diff
@@ -21,10 +21,12 @@
     },
     'UnixTimestampEncoder': {
         'missing_value_replacement': 'mean',
+        'datetime_format': False,
```

Comment:
Setting `datetime_format` to `False` does not learn the `datetimeformat` and helps us not having ot change all the integration tests.

Reply:
is there a way to instead figure out what the format string for the datetimes of the integration data would be and pass that in here instead? This is a strange workaround